src/handlers/settlement-windows.js

```diff
@@ -24,7 +24,7 @@ const handler = (router, routesContext) => {
         const settlementWindow = await routesContext.db.getSettlementWindowInfo(ctx.params.settlementWindowId);
 
         if (currentSettlementWindowId !== thisSettlementWindowId) {
-            const api = new Model({ endpoint: routesContext.config.settlementsEndpoint });
+            const api = new Model({ endpoint: routesContext.config.centralLedgerEndpoint });
```

Comment:
The lib called here explicitly states it's expecting the central settlement API not the central ledger API.
https://github.com/mojaloop/finance-portal-lib/blob/master/src/settlement/api.js#L20-L30
The `GET /settlements` endpoint doesn't exist on central ledger, either, I'm fairly sure.

Reply:
I expect perhaps instead the deploy config needs to change somewhere.

Reply:
It's trying to make a request to get the current open Window. This is an endpoint provided in the central ledger and with this change it works as expected. The settlements endpoint is the endpoint created on the finance portal story, about merging both the settlement-management and operator settlement services into one pod. Probably in some point these endpoints got mixed.

Reply:
Ok, you were right, my bad. sorry